e2e/specs/edit-existing-visit.spec.ts

```diff
@@ -122,3 +121,53 @@ test('Edit an existing ongoing visit to have an end time', async ({ page, api, p
     expect(updatedVisit.stopDatetime).not.toBeNull();
   });
 });
+
+test('Edit start date and time for a past visit', async ({ page, patient, pastVisit }) => {
+  const chartPage = new ChartPage(page);
+  const visitsPage = new VisitsPage(page);
+
+  const formattedStart = pastVisit.start.format('DD-MMM-YYYY');
+  const formattedEnd = pastVisit.end.format('DD-MMM-YYYY');
+
+  await test.step('When I Navigate to visits page for patient', async () => {
+    await visitsPage.goTo(patient.uuid);
+  });
+
+  await test.step('Then I click edit icn on past visit', async () => {
```

Comment:
Invalid usage of Gherkin syntax (check my comment here: https://github.com/openmrs/openmrs-esm-dispensing-app/pull/237#discussion_r2151971302):

This link might help clarify the syntax: https://cucumber.io/docs/gherkin/reference/

Reply:
Also, typo in `"icn"`